src/geometry.rs

```diff
@@ -91,33 +162,77 @@ impl Geometry {
         }
     }
 
-    /// Create new Plane with desired size.
-    pub fn plane(
-        sx: f32,
-        sy: f32,
-    ) -> Self {
+    /// Creates planar geometry in the XY plane.
+    ///
+    /// The `width` and `height` parameters specify the total length of the
+    /// geometry along the X and Y axes respectively.
+    ///
+    /// # Examples
+    ///
+    /// Unit square in the XY plane, centered at the origin.
+    ///
+    /// ```rust
+    /// # extern crate three;
+    /// fn make_square() -> three::Geometry {
+    ///     three::Geometry::plane(1.0, 1.0)
+    /// }
+    /// # fn main() { let _ = make_square(); }
+    /// ```
+    pub fn plane(width: f32, height: f32) -> Self {
```

Comment:
Did you run rustfmt?

Reply:
No, actually I undid the formatting for this case.

Reply:
Either we need to run rustfmt every time, or never run it.

Reply:
Fair enough.

Reply:
I think [`array-width`](https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#array_width) is not working correctly. It is set to 60 by default.

Reply:
Yep, looks like